Source/AwsCommonRuntimeKit/crt/Utilities.swift

```diff
@@ -130,13 +130,13 @@ extension aws_byte_cursor {
         }
 
         let data = Data(bytesNoCopy: self.ptr, count: self.len, deallocator: .none)
-        return String(data: data, encoding: .utf8)!
+        return String(decoding: data, as: UTF8.self)
     }
 
     func toOptionalString() -> String? {
         if self.len == 0 { return nil }
         let data = Data(bytesNoCopy: self.ptr, count: self.len, deallocator: .none)
-        return String(data: data, encoding: .utf8)!
+        return String(data: data, encoding: .utf8)
```

Comment:
This is used for the credential's access key, secret access key, and session token. We don't want partial characters in this case.